source/install/setting-up-local-machine-using-docker.rst

```diff
@@ -138,7 +134,7 @@ To delete your existing Docker preview and run a new version use:
 Accessing Your Container
 ------------------------
 
-If you wish to gain access to a shell on the container use:
+To access a shell on the container, run the following command:
```

Comment:
I would use **inside the container** instead of **on the container**. This matches the container terminology better.

Reply:
Updated! Thank you!

Reply:
Thanks!